sounddevice.py

```diff
@@ -2776,20 +2803,27 @@ def _split(value):
     return invalue, outvalue
 
 
-def _check(err, msg=""):
+def _check(err, msg=None):
```

Comment:
if default msg="" and no message is given you get: 

```PortAudioError: : Invalid number of channels [PaErrorCode xxx]```

With None, that extra semicolon + space doesn't get printed. I could also ignore it if bool(msg) evaluates to False, but I prefer not to assume anything about msg except that if it is None it should be ignored. In this particular case it is unlikely ever to be an issue, but I consider it good practice.